src/main/java/org/galatea/starter/domain/internal/StockPrices.java

```diff
@@ -7,27 +7,30 @@
 import javax.persistence.Id;
 import lombok.AllArgsConstructor;
 import lombok.Builder;
-import lombok.Getter;
-import lombok.Setter;
+import lombok.Data;
 import org.galatea.starter.domain.modelresponse.ResponsePrices;
 
-@Getter @Setter // Tells Lombok to generate getters/setters for all fields below
+@Data// Tells Lombok to generate getters/setters for all fields below
 @AllArgsConstructor //Tells java to instantiate all fields listed
 @Builder
 @Entity
-public class StockPrices extends ResponsePrices {
+public class StockPrices extends ResponsePrices  {
```

Comment:
don't think this should extend ResponsePrices. ResponsePrices is a nested class for the JSON/AlphaVantage representation. this is our internal model.